apps/web/components/eventtype/EventLimitsTab.tsx

```diff
@@ -145,12 +148,20 @@ export const EventLimitsTab = ({ eventType }: Pick<EventTypeSetupProps, "eventTy
     defaultValue: periodType?.type,
   });
 
+  const { shouldLockIndicator, shouldLockDisableProps } = lockedFieldsManager(
+    eventType,
+    t("locked_fields_description")
+  );
+
   return (
     <div className="space-y-8">
       <div className="space-y-4 lg:space-y-8">
         <div className="flex flex-col space-y-4 lg:flex-row lg:space-y-0 lg:space-x-4">
           <div className="w-full">
-            <Label htmlFor="beforeBufferTime">{t("before_event")} </Label>
+            <Label htmlFor="beforeBufferTime">
+              {t("before_event")}
+              {shouldLockIndicator("bookingLimits")}
```

Comment:
This gets the locked indicator if the passed field name is actually locked, otherwise it doesn't return an object hence nothing is shown